package/src/numtools/distances.pyx

```diff
@@ -255,3 +256,56 @@ def triclinic_pbc(c_numpy.ndarray coords,
 
     _triclinic_pbc(<coordinate*> coords.data, numcoords,
                    <coordinate*> box.data, <float*>box_inverse.data)
+
+
+@cython.boundscheck(False)
+def contact_matrix_no_pbc(coord, sparse_contacts, cutoff):
+    cdef int rows = len(coord)
+    cdef double cutoff2 = cutoff ** 2
+    cdef float[:, ::1] xyz = coord
+
+    cdef int i, j
+    cdef double x, y, z, dist
+    for i in range(rows):
+        sparse_contacts[i, i] = True
+        for j in range(i+1, rows):
+            x = xyz[i, 0] - xyz[j, 0]
+            y = xyz[i, 1] - xyz[j, 1]
+            z = xyz[i, 2] - xyz[j, 2]
+            dist = x*x + y*y + z*z
+            if dist < cutoff2:
+                sparse_contacts[i, j] = True
+                sparse_contacts[j, i] = True
+
+
+@cython.boundscheck(False)
+def contact_matrix_pbc(coord, sparse_contacts, box, cutoff):
+    cdef int rows = len(coord)
+    cdef double cutoff2 = cutoff ** 2
+    cdef float[:, ::1] xyz = coord
+    cdef float[::1] box_view = box
+    cdef float[::1] box_half = box / 2.
+
+    cdef int i, j
+    cdef double x, y, z, dist
+    for i in range(rows):
+        sparse_contacts[i, i] = True
+        for j in range(i+1, rows):
+            x = xyz[i, 0] - xyz[j, 0]
+            y = xyz[i, 1] - xyz[j, 1]
+            z = xyz[i, 2] - xyz[j, 2]
+
+            if abs(x) > box_half[0]:
```

Comment:
Good catch.  I always assume that particles are in one of the neighboring images and then wonder that happened when this is not the case.  I will first add this to the unit-tests and then fix it.